Program.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+
+namespace Зачёт_В25
+{
+    class MainClass
+    {
+        public static void Main(string[] args)
+        {
+
+
+            Console.WriteLine("Проверка работы функции S(n):");
+            long sn1 = 1;
+            for (long i = 2; i <= 6; i++)
+            {
+                sn1 += Qn(i);
+                Console.WriteLine(i + " - " + sn1);
+            }
+            Console.WriteLine("Выполнение программы:");
+            long sn = 1;
+            for (long i = 2; i <= 1000000; i++)
+            {
+                sn += Qn(i);
+            }
```

Comment:
Лишние фигурные скобки. Если тело цикла состоит из одного оператора, фигурные скобки можно не ставить.
```suggestion
            for (long i = 2; i <= 1000000; i++)
                sn += Qn(i);
```